content/docs/guides/ha_scale/perf-scale.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Performance and Scalability"
+description: "Performance benchmark on control plane and data plane sidecar and scalability test of Open Service Mesh"
+type: docs
+---
+
+Open Service Mesh aims to create a high-performance service mesh while providing all the essential service mesh features. This document describes the tests performed to evaluate the performance and scalability of OSM.
+
+## Testing environment
+
+This test is performed on a Kubernetes cluster provisioned and managed by [Microsoft Azure Kubernetes Service](https://azure.microsoft.com/en-us/services/kubernetes-service/). The cluster has 100 Standard_DS2_v2 nodes (each node has 2 vCPUs and 7GB memory). It is ensured that there is only one OSM controller pod running with default resource quota during the test. OSM is configured in non-permissive traffic mode.
+
+## Process
```

Comment:
maybe note that we are using http 1 for this, since http 2 or grpc would have different performance characteristics)

Reply:
Sure.

Yes we should mentioned this is a minimal setup of the mesh. We will benchmark other popular features of OSM and service mesh in general in the future.